CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.0.2 (Apr 11, 2022)
```

Comment:
Is this the correct version number

Reply:
master had 6.0.1 ... so incremented to 6.0.2 for this bug fix.

Reply:
Referenced: https://www.terraform.io/plugin/sdkv2/best-practices/versioning#versioning-specification
Since the issue was seen as bug … i did a PATCH increment (6.0.1 -> 6.0.2)

Reply:
Per https://www.terraform.io/plugin/sdkv2/best-practices/versioning#example-minor-number-increments

> Implementing **new attributes** within the provider configuration or **an existing resource** or data source

(emphasis mine)

So I think the version should be a minor increment, i.e. 6.1.0.

Reply:
ok